src/main/features/eligibility/views/mcol-eligibility.njk

```diff
@@ -0,0 +1,41 @@
+{% extends "layout.njk" %}
+{% from "./macro/mcolSupport.njk" import mcolSupportWidget %}
+
+{% set title = t('Check you can use Money Claim Online | Money claims') %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-large">{{ t('Check you can use Money Claim Online (MCOL)') }}</h1>
+
+      <p>{{ t('You can use the existing MCOL service to claim:') }}</p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('for a fixed sum under £100,000') }}</li>
+        <li>{{ t('against no more than 2 people or organisations') }}</li>
+      </ul>
+
+      <p>
+        {{ t('You can use the service from anywhere in the UK, but only against someone with an address in England or Wales.') }}
+      </p>
+
+      <p>{{ t('You must be 18 or over to make a claim.') }}</p>
+
+      <h1 class="heading-medium">{{ t('What you can’t claim for on MCOL') }}</h1>
```

Comment:
Please do not use `h1` again, it should be `h2`. Content on the page should have some hierarchy, headlines are the tool to implement that hierarchy.

Reply:
addressed in latest commits

Reply:
looks like not pushed